.gitlab/corona-jobs.yml

```diff
@@ -1,21 +1,16 @@
 #############################################################################
-## Copyright (c) 2016-21, Lawrence Livermore National Security, LLC
-## and RAJA project contributors. See the RAJA/COPYRIGHT file for details.
-##
-## SPDX-License-Identifier: (BSD-3-Clause)
-##############################################################################
-
-hip_4_1_gcc_8_1_0:
-  variables:
-    SPEC: "+hip~openmp %gcc@8.1.0 cxxflags='-finline-functions -finline-limit=20000' cflags='-finline-functions -finline-limit=20000' ^hip@4.1.0"
-  extends: .build_and_test_on_corona
+# Copyright (c) 2016-22, Lawrence Livermore National Security, LLC
+# and RAJA project contributors. See the RAJA/LICENSE file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+#############################################################################
 
-hip_4_1_clang_9_0_0:
+hip_4_5_2_clang_13_0_0 (build and test on corona):
   variables:
-    SPEC: "+hip~openmp %clang@9.0.0 cxxflags=--gcc-toolchain=/usr/tce/packages/gcc/gcc-8.1.0 ^hip@4.1.0"
+    SPEC: "+rocm~openmp amdgpu_target=gfx906 %clang@13.0.0 cxxflags=--offload-arch=gfx906 ^blt@develop ^hip@4.5.2"
   extends: .build_and_test_on_corona
 
-hip_4_2_gcc_8_1_0:
-  variables:
-    SPEC: "+hip~openmp %gcc@8.1.0 cxxflags='-finline-functions -finline-limit=20000' cflags='-finline-functions -finline-limit=20000' ^hip@4.2.0"
-  extends: .build_and_test_on_corona
+#hip_4_5_2_clang_13_0_0_desul_atomics (build and test on corona):
```

Comment:
Should we add back in this hip job?

Reply:
Not yet for the Perf Suite

Reply:
Ok. Also, why the change from clang 11 to clang 9 in some of the lassen jobs?

Reply:
which jobs? I switched from clang-ibm to regular clang b/c I think there is a bug in the runtime (I'm going to talk to John G. about that). Otherwise, I didn't change any versions, except for clang-ibm 10 --> clang 11 since there is no clang 10 in the version of the radiuss configs we are using.

Reply:
(I started a new conversation thread for this)